tests/ui/repr/repr_align_greater_usize.rs

```diff
@@ -0,0 +1,26 @@
+//@ revisions: msp430 aarch32
+//@[msp430] check-fail
```

Comment:
Suggestion: this is redundant for ui tests, ui tests are `check-fail` by default.